sass/components/list.sass

```diff
@@ -0,0 +1,26 @@
+$list-background-color: $white !default
+$list-radius: $radius !default
+
+$list-item-border: 1px solid $border !default
+$list-item-color: $text !default
+$list-item-active-color: $primary !default
+$list-item-active-text-color: $white !default
+$list-item-hover-color: $background !default
+
+.list
+  background-color: $list-background-color
+  box-shadow: $card-shadow
+  border-radius: $list-radius
+  &.is-hoverable > .list-item:hover:not(.is-active)
+    background-color: $list-item-hover-color
+    cursor: pointer
+
+.list-item
+  color: $list-item-color
+  display: block
+  padding: 0.375rem 0.75rem
```

Comment:
You need `padding: 0.5em 1em` here.

Reply:
@jgthms Why is the `0.5em 1em` padding better than `0.375rem 0.75rem`?

Reply:
Uses `em`so it's proportional to the list's font size, and just looks better.

Reply:
I get the second part, but don't understand how do you decide whether to use `rem` or `em`.
For vertical margins(level, card, panel) you tend to use `rem` but sometimes(form label) you use `em`.
For horizontal margins you tend to use `em` but sometimes(tags) you use `rem`.
Similarly for paddings.
What's your rule of thumb when it comes to these units?

I feel like I'm being annoying but I just want to understand your train of though.

Reply:
Margin is between elements, hence a common denominator like rem. Padding is within an element and the visual balance is assured when everything is proportional to the font size hence em.

Button padding is em. Same for form controls.